blog/2021-12-20-openhab-3-1-release.md

```diff
@@ -0,0 +1,79 @@
+---
```

Comment:
Please rename this file from `2021-12-20-openhab-3-1-release.md` to `2021-12-20-openhab-3-2-release.md`!

Reply:
sure